packages/components/src/internal/components/domainproperties/models.tsx

```diff
@@ -330,6 +331,11 @@ export class DomainDesign
                     const valueType = typeof rawVal;
                     let value = valueIsEmpty(rawVal) ? '' : rawVal;
 
+                    // Since rangeURI is not set on field creation, pull rangeURI value from dataType
+                    if (key === 'rangeURI' && value === '') {
+                        value = dataType.rangeURI;
+                    }
```

Comment:
Is it possible to add a quick jest test for this case?

Reply:
Added check of rangeURI on a minimal field object within DomainPropertiesGrid.spec.tsx! 👍